src/DevYeah.LMS/DevYeah.LMS.Web/app/public/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <!--
```

Comment:
Should add 

`<meta name="description" content="ALL_THE_KEYWORDS_HERE">`

 for SEO